synthea-hiv/uploader/main.py

```diff
@@ -107,99 +106,37 @@ def list_all_files(directory: str) -> Dict[str, Set[pathlib.PosixPath]]:
 
 
 def convert_to_bundle(json_file: pathlib.PosixPath) -> bundle.Bundle:
-  """Loops through list of JSON files, and loads content of each file to create Bundle object.
+  """Loads content of file to create Bundle object.
 
   Args:
    json_file: Set containing JSON files that need to be read
 
   Returns:
-    List containing Bundle objects
+    Bundle object
   """
   with open(json_file) as f:
     data = json.loads(f.read())
     return bundle.Bundle(json_file, data)
 
-
-def fetch_location(sink: fhir_client.FhirClient) -> Dict[str, str]:
```

Comment:
moved this to uploader.py as a method.I wanted all calls using FhirClient to be in one module and not across multiple ones

Reply:
Let's separate this into another PR; this is really not related to new image creation.